g3doc/get_started.md

```diff
@@ -78,7 +78,7 @@ TFMA can perform large-scale distributed evaluation of your model by using
 evaluation results can be visualized in a Jupyter notebook using the frontend
 components included in TFMA.
 
-![TFMA Slicing Metrics Browser](/images/tfma-slicing-metrics-browser.png)
+![TFMA Slicing Metrics Browser](/g3doc/images/tfma-slicing-metrics-browser.png)
```

Comment:
How about using a relative path: `images/tfma-slicing-metrics-browser.png` instead?

Reply:
I see, I'll fix. thank you! 😄